Automattic/wp-calypso
# Launchpad: Make sidebar strings dynamic and translated

### Proposed Changes

This PR ensures strings in the Launchpad sidebar are translated, taking into account that strings may dynamically vary depending on the Launchpad flow (ie, Newsletter vs Link in Bio). It also removes some used, nontranslated strings from task.tsx. 

### Testing Instructions

Note: We can't accurately test actual translations until we have translations. So these testing instructions focus on ensuring that English language strings still show correctly, and that strings adjust dynamically based on flow type. 

1. Check out branch. Run `yarn` and `yarn start` if you haven't already.
2. Load the following URL and confirm that "Newsletter" appears in the top left of the screen; that other strings on the sidebar read correctly; and that the "Voila" headline references "Newsletter".
3. Load the following URL and confirm that "Link in Bio" appears in the top left of the screen; that other strings on the sidebar read correctly; and that the "Voila" headline references "Link in Bio".

Related to https://github.com/Automattic/wp-calypso/issues/66757


UPDATE/ADDITION FOR TRANSLATORS: I received a note about adding screenshots to help translators understand the context of strings. So I just wanted to add a screenshot and other context. 

All the new strings in this PR appear on a single screen - the Launchpad screen which is part of a new onboarding flow. This screen is always the same, but some of the strings change depending on the type of site - newsletter, link in bio, podcast. 

Below is a screenshot of the Launchpad screen for a Launch in Bio site. If this were for Newsletter or Podcast, then each occurrence of "Launch in Bio" would be replaced with "Newsletter" or "Podcast". 

Hope that's helpful!

<img width="1508" alt="Screenshot on 2022-08-24 at 09-55-19" src="https://user-images.githubusercontent.com/21228350/186465332-181100a0-5673-4c93-82a6-dfb14e487079.png">
